Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -190,9 +190,9 @@ skipButtonConfig state =
         , margin = MarginTop 22
         , id = "SkipButton"
         , enableLoader = (JB.getBtnLoader "SkipButton")
-        , visibility = if not issueFaced || state.data.ratingViewState.doneButtonVisibility then VISIBLE else GONE
-        , isClickable = issueFaced || state.data.ratingViewState.selectedRating > 0
-        , alpha = if issueFaced || (state.data.ratingViewState.selectedRating >= 1) then 1.0 else 0.4
+        , visibility = if state.data.ratingViewState.doneButtonVisibility || not state.props.showOfferedAssistancePopUp then VISIBLE else GONE
+        , isClickable = issueFaced || state.data.ratingViewState.selectedRating > 0 || state.data.ratingViewState.selectedYesNoButton > -1
```

Comment:
Can you revisit these all conditions 
Also Don't you think  `>=0` will be more clear than ` > -1`

Reply:
done